config/tables/plans.json

```diff
@@ -22,8 +22,16 @@
       ],
       "Projection": {
         "ProjectionType": "INCLUDE",
-        "NonKeyAttributes": "id,systemIds"
+        "NonKeyAttributes": ["id", "systemIds"]
+      },
+      "ProvisionedThroughput": {
+        "ReadCapacityUnits": 1,
+        "WriteCapacityUnits": 1
```

Comment:
This is required due to a weird bug when creating the table in CircleCI, where even though we are `PAY_PER_REQUEST` it was requiring these values. We can investigate and fix in the future.